test/files/neg/ordering-migration.scala

```diff
@@ -1,4 +1,4 @@
-// scalac: -deprecation -Xfatal-warnings
+// scalac: -Xmigration -Werror
```

Comment:
should this file be called `t11844.scala` instead?

Reply:
I generally prefer English name for partests since it makes it easier to figure out what it's testing afterwards, and you can also grep easier from sbt shell.

Reply:
If you're going to move it, move it to `pos`, seeing as it's no longer a negative test.